src/miner.h

```diff
@@ -87,7 +87,8 @@ struct CompareTxIterByAncestorCount {
     {
         if (a->GetCountWithAncestors() != b->GetCountWithAncestors())
             return a->GetCountWithAncestors() < b->GetCountWithAncestors();
-        return CTxMemPool::CompareIteratorByHash()(a, b);
+        // SYSCOIN
+        return a->GetTime() < b->GetTime();
     }
```

Comment:
This is a critical improvement by removing ordering by time in graph.cpp we simply modify the comparator to return time ordering if ancestor counts are equal, we want ancestor ordering to be higher prescedence anyways followed by time so this shoudl work for us.